src/core/issues-notes.js

```diff
@@ -203,6 +193,30 @@ function createIssueSummaryEntry(l10nIssue, report, id) {
   `;
 }
 
+/**
+ *
+ * @param {*} issueList
+ */
+function makeIssueSectionSummary(issueList) {
+  const issueSummaryElement = document.getElementById("issue-summary");
+
+  if (!issueSummaryElement) return;
+  const heading = issueSummaryElement.querySelectorAll("h2, h3, h4, h5, h6");
```

Comment:
I think here you just want whatever the first one is. The assumption being that there is a heading in the section, which is fine. 
 
```suggestion
  const heading = issueSummaryElement.querySelector("h2, h3, h4, h5, h6");
```